parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -25,6 +25,47 @@ installDeps() {
     fi
 }
 
+installKataDeps() {
+    if [[ $OS_VERSION == "2.0" ]]; then
+      for dnf_package in cargo opa parted qemu-img moby-runc python3-devel python3-pip kernel-mshv cloud-hypervisor kata-containers moby-containerd-cc mshv-bootloader mshv-linuxloader mshv; do
+        if ! dnf_install 30 1 600 $dnf_package; then
+          exit $ERR_APT_INSTALL_TIMEOUT
+        fi
+      done
+
+      echo "Install UVM build pipeline artifacts"
+      mkdir -p /opt/confidential-containers/share/kata-containers/
+      mv igvm-debug.bin /opt/confidential-containers/share/kata-containers/igvm-debug.bin
+      mv igvm-measurement /opt/confidential-containers/share/kata-containers/igvm-measurement
+      mv kata-containers-initrd.img /opt/confidential-containers/share/kata-containers/kata-containers-initrd.img
+
+      echo "TEMP: install cloud-hypervisor-igvm"
+      mkdir -p /opt/confidential-containers/bin/
+      mv cloud-hypervisor-igvm /opt/confidential-containers/bin/cloud-hypervisor-igvm
+      chmod 755 /opt/confidential-containers/bin/cloud-hypervisor-igvm
```

Comment:
Is this chmod required, it seems that this will be handled in packer_source.sh's L130, `cpAndMode $KATA_CLH_SRC $KATA_CLH_DEST 0755`

Reply:
No, it's not. Actually, these `mv` commands are not required neither. Just updated packer_source.sh to copy binaries directly into the confidential-containers directory.